static/metricsgraphics/socorro-lens.html

```diff
@@ -0,0 +1,240 @@
+          width: 300,
+          height: 170,
+          target: document.getElementById('chart'),
+          x_accessor: 'date',
+          y_accessor: channel,
+          yax_count: 3,
+          chart_type: "line",
+          mouseover: globals.mouseover
+        });
+        if (globals.url.length > 1) {
+          var target = document.getElementById('warn');
+          target.style.visibility = "visible";
+          target.innerHTML = "Report will open " + globals.url.length + " tabs</p>";
+        }
+        var mouseouts = d3.selectAll('.mg-rollover-rect rect').on('mouseout');
+        d3.selectAll('.mg-rollover-rect rect').on('click', function (d) {
+          $.each(globals.url, function (i) {
+            window.open(globals.url[i], '_blank');
+          });
+        });
+      }
+
+      function redraw(channel) {
+        MG.data_graphic({
+          data: items,
+          width: 300,
+          height: 170,
+          target: document.getElementById('chart'),
+          x_accessor: 'date',
+          y_accessor: channel,
+          yax_count: 3,
+          chart_type: "line",
+          mouseover: globals.mouseover
+        });
+
+        if (globals.url.length > 1) {
+          var target = document.getElementById('warn');
+          target.style.visibility = "visible";
+          target.innerHTML = "[!] Report will open " + globals.url.length + " tabs due to signature length.</p>";
+        }
+
+        var mouseouts = d3.selectAll('.mg-rollover-rect rect').on('mouseout');
+        d3.selectAll('.mg-rollover-rect rect').on('click', function (d) {
+          $.each(globals.url, function (i) {
+            window.open(globals.url[i], '_blank');
+          });
+        });
+      }
+
+      function loadGraph(search, match) {
+        // Get all signatures from the Bugzilla page
+        var signatures = getSignaturesFromURL(search, match);
+        // Initialize chart data
+        items = [];
+        for (var i = 1; i < 181; i++) {
+          items.push({
+            "date": convertDate(new Date((new Date()).valueOf() - 1000 * 60 * 60 * 24 * i)),
+            "release": 0,
+            "beta": 0,
+            "nightly": 0,
+            "esr": 0,
+            "all": 0
+          });
+        }
+
+        // Process the Socorro data
+        if (items.length >= 180) {
+          var processed = [0, 0];
+          var processed_groups = 0;
+          $.each(signatures, function (i) {
+            var processed_data = 0;
+            // Set the report URL, this will be used to load the data report on click
+            if (!globals.url[i]) globals.url[i] = "";
+            globals.url[i] = globals.url_base + signatures[i];
+            // Iterate through the Socorro data and create the chart data object
+            var url = "https://crash-stats.mozilla.com/api/SuperSearch/?" + signatures[i] + "&date=%3E%3D" + start_date + "&date=%3C%3D" + end_date + "&_histogram.date=release_channel&_histogram_interval=1d&_results_number=0";
```

Comment:
Can this URL be moved into a config file easily? (If not, no worries.)

Reply:
I don't know, perhaps Dylan can advise.